### PR TITLE
added run name to the success popup

### PR DESCRIPTION
**Pull request by dwagg**

### CODE REVIEW

File: Mobile Test Runner for Jama/Mobile Test Runner for Jama/TestListViewController.swift [hunk 1]
```diff
@@ -59,7 +60,7 @@ class TestListViewController: UIViewController {
         //If a test run has just updated before this view appeared
         if displayTestRunAlert {
             //Alert the user that the update was successful
-            let updateSuccededAlert = UIAlertController(title: "Run updated", message: "Test run was successfully committed.", preferredStyle: UIAlertControllerStyle.alert)
+            let updateSuccededAlert = UIAlertController(title: updatedRunName + " updated", message: updatedRunName + " was successfully committed.", preferredStyle: UIAlertControllerStyle.alert)
```
Comment (benleelawrence) [inline Mobile Test Runner for Jama/Mobile Test Runner for Jama/TestListViewController.swift]: Succeed misspelled

File: Mobile Test Runner for Jama/Mobile Test Runner for Jama/TestListViewController.swift [hunk 2]
```diff
@@ -59,13 +60,13 @@ class TestListViewController: UIViewController {
         //If a test run has just updated before this view appeared
         if displayTestRunAlert {
             //Alert the user that the update was successful
-            let updateSuccededAlert = UIAlertController(title: "Run updated", message: "Test run was successfully committed.", preferredStyle: UIAlertControllerStyle.alert)
+            let updateSucceededAlert = UIAlertController(title: updatedRunName + " updated", message: updatedRunName + " was successfully committed.", preferredStyle: UIAlertControllerStyle.alert)
```
Comment (dwagg) [inline Mobile Test Runner for Jama/Mobile Test Runner for Jama/TestListViewController.swift]: I am not sure if we want the run name in the title and body, just the title or just the body. I went with both but I am not 100% sure that is the right way to go. Especially when there are long run names.

**Comment by benleelawrence** [comment]
Before this update I kind of thought the success popup stayed onscreen for too long, but now for really long test run names we might want to add another second.  Another idea would be to only display the test run name once, e.g., title: " Test Run updated", message: updatedRunName + " was successfully committed." or the other way around

**Comment by dwagg** [comment]
Do we want to move away from a timer and just put a dismiss button on the popup? That way the user decides how long the popup sticks around.

**Comment by lcc4** [comment]
I like Ben's idea of the popup containing the message:  
" Test Run updated", message: updatedRunName + " was successfully committed."

And allowing the user to dismiss the popup. Want to try implementing those changes?

**Review by lcc4 — CHANGES_REQUESTED** [review]
Please modify the popup per Ben's suggestions.

**Review by lcc4 — APPROVED** [review]

**Pull request merged**